cl.conf

```diff
@@ -133,13 +133,15 @@ storage:
           sudo curl -Lo $MK https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && sudo chmod +x $MK
           sudo curl -Lo $KUBECTL https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && sudo chmod +x $KUBECTL
 
+          cp -af $KUBECTL $USER_HOME/.bricks/bin/kubectl
+
           export MINIKUBE_WANTUPDATENOTIFICATION=true
           export MINIKUBE_WANTREPORTERRORPROMPT=false
           export MINIKUBE_WANTKUBECTLDOWNLOADMSG=false
           export MINIKUBE_HOME=$USER_HOME/.bricks/clusters/vagrant
           export CHANGE_MINIKUBE_NONE_USER=true
 
-          export KUBECONFIG=$HOME/vagrant/kubeconfig
+          export KUBECONFIG=$HOME/vagrant/kubeconfig.yaml
```

Comment:
How does this file get created? Does minikube look at the KUBECONFIG variable and store the contents of the configuration in the specified file?

Reply:
I think so, it just stores the config kvs for minikube

```
core@core-01 ~ $ cat $HOME/.kube/config
apiVersion: v1
clusters:
- cluster:
    certificate-authority: /Users/rm/.bricks/clusters/vagrant/.minikube/ca.crt
    server: https://127.0.0.1:8443
  name: minikube
contexts:
- context:
    cluster: minikube
    user: minikube
  name: minikube
current-context: minikube
kind: Config
preferences: {}
users:
- name: minikube
  user:
    as-user-extra: {}
    client-certificate: /Users/rm/.bricks/clusters/vagrant/.minikube/client.crt
    client-key: /Users/rm/.bricks/clusters/vagrant/.minikube/client.key
```

Reply:
If you do 'kubectl get po' on core-01 host with this change, it still work fine?

Reply:
yeah